geoportal/c2cgeoportal_geoportal/views/entry.py

```diff
@@ -329,7 +329,7 @@ def _create_layer_query(role_id, version, interface):
             query = models.DBSession.query(main.LayerV1.name)
         else:
             query = models.DBSession.query(main.Layer.name).filter(
-                main.Layer.item_type.in_(["l_wms", "l_wmts"])
+                ~main.Layer.item_type.in_(["layerv1"])
```

Comment:
For me this is more readable :-)
```suggestion
                main.Layer.item_type.notin_(["layerv1"])
```